lib/blueprinter/extractors/auto_extractor.rb

```diff
@@ -2,7 +2,8 @@ module Blueprinter
   class AutoExtractor < Extractor
     def extract(field_name, object, local_options, options = {})
       extractor = object.is_a?(Hash) ? HashExtractor : PublicSendExtractor
-      extractor.extract(field_name, object, local_options, options)
+      extraction = extractor.extract(field_name, object, local_options, options)
+      options.key?(:datetime_format) ? extraction.strftime(options[:datetime_format]) : extraction
```

Comment:
What about when extraction is not a DateTime object?
Do we want to surface our own error in that case? i.e. something like: `"Attempting to format invalid DateTime object"`

Reply:
Makes sense. Done 👍